src/model/model.ts

```diff
@@ -232,12 +228,7 @@ export class Model {
   }
 
   // removeFilteredPolicy removes policy rules based on field filters from the model.
-  public removeFilteredPolicy(
-    sec: string,
-    key: string,
-    fieldIndex: number,
-    ...fieldValues: string[]
-  ): boolean {
+  public removeFilteredPolicy(sec: string, key: string, fieldIndex: number, ...fieldValues: string[]): boolean {
```

Comment:
I already fixed this in: https://github.com/casbin/node-casbin/commit/7e78cbaa9b4461cb7f241743fa83c4afcd107be8. Please remove it.